ansible/roles/airflow-config/templates/airflow.cfg.j2

```diff
@@ -1038,7 +1038,7 @@ max_dagruns_per_loop_to_schedule = 20
 # Should the Task supervisor process perform a "mini scheduler" to attempt to schedule more tasks of the
 # same DAG. Leaving this on will mean tasks in the same DAG execute quicker, but might starve out other
 # dags in some circumstances
-schedule_after_task_execution = True
+schedule_after_task_execution = False
```

Comment:
Do we know what was the effect of this change?

Reply:
Having that parameter set means that the same process can take on another task from the same dag, which can speed things up a bit, but [folklore](https://stackoverflow.com/a/70128529) says that that can also contribute to the SIGTERMs when a process with a completed task is killed when another task is still underway. How much it contributed in our case is hard to say though as the effect is stochastic and we didn't carry out comparisons of long runs with single parameters tweaked. But at least I haven't observed any new problems.